src/h5web/visualizations/heatmap/ColorBar.tsx

```diff
@@ -1,29 +1,28 @@
 import React from 'react';
 import { AxisRight } from '@vx/axis';
 import { useMeasure } from 'react-use';
-import shallow from 'zustand/shallow';
 import { adaptedNumTicks } from '../shared/utils';
 import { useInterpolator } from './hooks';
-import { useHeatmapConfig } from './config';
 import styles from './HeatmapVis.module.css';
 import { generateCSSLinearGradient } from './utils';
-import { SCALE_FUNCTIONS } from '../shared/models';
-
-function ColorBar(): JSX.Element {
-  const [dataDomain, customDomain, scaleType] = useHeatmapConfig(
-    (state) => [state.dataDomain, state.customDomain, state.scaleType],
-    shallow
-  );
```

Comment:
I like the idea of using less of `useHeatmapConfig` so we can pass the config as a prop to `HeatmapVis`. That being said, this will trigger more re-renders of `HeatmapVis` itself, so we may have to add `React.memo()` here and there, like on `VisCanvas` to avoid re-rendering too many children when the user changes the color map or whatever.

Reply:
I've opened an issue to remember.

Reply:
All right. I did not notice any performance drop but it is better to keep it in mind